cmd/pd-server/main.go

```diff
@@ -40,8 +41,27 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	serviceMode    = "service"
+	tsoServiceMode = "tso-service"
```

Comment:
nit: can we combine these two parameters into 'service-mode-tso'? This is the subcommand format that I know. If "service tso-service" is TiDB convention, please ignore my comment.